angular-beans/pom.xml

```diff
@@ -5,22 +5,34 @@
 	<artifactId>angularBeans</artifactId>
 	<version>1.0.3-SNAPSHOT</version>
 	<name>AngularBeans</name>
+	<description>AngularBeans Framework</description>
+	<url>http://bessemhmidi.github.io/AngularBeans</url>
+	
+	<scm>
+		<url>https://github.com/bessemHmidi/AngularBeans.git</url>
+	</scm>
+	
+	<licenses>
+		<license>
+			<name>LGPL</name>
+			<distribution>repo</distribution>
+			<url>http://www.gnu.org/licenses/lgpl-3.0.html</url>
+		</license>
+	</licenses>
+	
+	<developers>
+		<developer>
+			<name>Bessem Hmidi</name>
+			<email>bessem.hmidi.perso@gmail.com</email>
+		</developer>
+	</developers>
 
 	<properties>
 		<maven.compiler.source>1.7</maven.compiler.source>
 		<maven.compiler.target>1.7</maven.compiler.target>
-
-		<!-- <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding> -->
```

Comment:
True, I removed this line during code cleanup because we've not been using utf-8 anyway, what's the point of putting a commented tag ? I know It would be better to use UTF-8 to avoid platform-dependency issues but no one claimed (until now haha)